src/ape/contracts/base.py

```diff
@@ -463,6 +463,19 @@ def __len__(self):
         logs = self.provider.get_contract_logs(self.log_filter)
         return sum(1 for _ in logs)
 
+
+    def __call__(self, *args: Any, **kwargs: Any) -> "BaseContractLog":
+        event_args = dict(zip([input.name for input in self.abi.inputs], args))
+        event_args.update(kwargs)
+
+        log_data = {
+            field: event_args.get(field, None) for field in ContractLog.__fields__.keys()
+        }
+        log_data["address"] = self.contract.address
+        log_data["abi"] = self.abi
+
+        return MockContractLog(**log_data)
```

Comment:
Or could do this instead
```suggestion
        return MockContractLog(
            abi=self.abi,
            address=self.contract.address,
            event_arguments=event_args,
        )
```